xmtp_mls/src/lib.rs

```diff
@@ -54,16 +59,32 @@ pub trait Delete<Model> {
 
 #[cfg(test)]
 mod tests {
-    use std::sync::Once;
+    use tracing_test::traced_test;
 
-    static INIT: Once = Once::new();
-
-    /// Setup for tests
+    // Execute once before any tests are run
     #[ctor::ctor]
+    // Capture traces in a variable that can be checked in tests, as well as outputting them to stdout on test failure
+    #[traced_test]
     fn setup() {
-        INIT.call_once(|| {
-            tracing_subscriber::fmt::init();
-        })
+        // Capture logs (e.g. log::info!()) as traces too
+        let _ = tracing_log::LogTracer::init();
+    }
```

Comment:
Heads up @insipx - this allows tests to capture the logs and check them, while still outputting them to stdout

Reply:
sounds awesome!